### ISSUE TITLE
About pre-trained model

### ISSUE THREAD

**Issue by FriedRonaldo** [Mark12Ding/FAME#2]
Hi, thanks for the interesting work!

I run the evaluation of the pre-trained model of I3D on UCF-101, I got 88.918% of top-1 accuracy which is different from the record in the paper (88.6%) when I used just two GPUs (V100).

Can you check if the provided ckpt file contains the weight of the model reported?

And could you provide the ckpt files of pre-trained models? I mean R(2+1)D on Kinetics-400, UCF-101 and HMD51, I3D on HMD51.

Thanks!

**Comment by Mark12Ding** [comment]
Thanks for the attention. The checkpoints will be released when ready.

**Comment by nanyyyyyy** [comment]
The download link has expired. Would you mind sharing it again? Thank you so much!
